### ISSUE TITLE
Add test coverage for `<DescriptionList`

### ISSUE THREAD

**Issue by dexterca** [lifeomic/chroma-react#362]

**Comment by shawnzhu** [comment]
let me see

**Comment by shawnzhu** [comment]
## Now

on feature branch `feat-description-list`, the current code coverage:

```
---------------------------------------|---------|----------|---------|---------|---------------------
File                                   | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s   
---------------------------------------|---------|----------|---------|---------|---------------------
All files                              |    8.94 |     3.24 |   13.65 |    8.01 |                     
 components/DescriptionList            |   80.74 |    43.75 |    87.1 |   80.95 |                     
  DescriptionDetails.tsx               |     100 |       75 |     100 |     100 | 126-134,182         
  DescriptionDivider.tsx               |   54.55 |      100 |       0 |      60 | 9,45-48             
  DescriptionList.tsx                  |   96.61 |       50 |     100 |     100 | 16-17,180-211       
  DescriptionListGroupHeading.tsx      |     100 |      100 |     100 |     100 |              
```

## Expected

100%

**Comment by shawnzhu** [comment]
## Progress

w/ #366:

```
---------------------------------------|---------|----------|---------|---------|---------------------
File                                   | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s   
---------------------------------------|---------|----------|---------|---------|---------------------
 components/DescriptionList            |   98.52 |     57.5 |     100 |     100 |                     
  DescriptionDetails.tsx               |     100 |       75 |     100 |     100 | 126-134,182         
  DescriptionDivider.tsx               |     100 |      100 |     100 |     100 |                     
  DescriptionList.tsx                  |   96.61 |       55 |     100 |     100 | 16-17,196-211       
  DescriptionListGroupHeading.tsx      |     100 |      100 |     100 |     100 |                     
  DescriptionTerm.tsx                  |     100 |       45 |     100 |     100 | 126-222
```

Those missing branches are custom controls of `<DescriptionDetails/>` and `<DescriptionTerm/>`

**Comment by shawnzhu** [comment]
closed by #366
